src/background/tab-state.js

```diff
@@ -160,8 +164,23 @@ export default function TabState(initialState, onchange) {
   this.updateAnnotationCount = async function (tabId, tabUrl) {
     const INITIAL_WAIT_MS = 1000;
     const MAX_WAIT_MS = 3000;
+    const EXPIRATION_CACHE_MS = 3000;
```

Comment:
```suggestion
    const CACHE_EXPIRATION_MS = 3000;
```

Suggested change of word order to be more consistent with the other constants here.